index.html

```diff
@@ -8642,12 +8663,12 @@ <h3 id="Nature"><span class="wip">Nature - Use with Caution</span></h3>
                </td>
                <td>Parasitic Coating
                </td>
-               <td>Every time an ally within 25ft damages a distinct enemy they heal for
+               <td>Every time an ally within 20ft damages a distinct enemy they heal for
```

Comment:
This could get very spicy with AoEs, maybe a bit toooooo generically good but we can wait and see. Might just need a cap on it of some amount to not be ridiculous and broken - especially as it effects many allies, if it was just a single ally that might also be more reasonable.